### ISSUE TITLE
Library switching in the NuGet

### ISSUE THREAD

**Issue by maluoi** [StereoKit/StereoKit#417]
## Describe the feature

The ability to switch out which StereoKitC or OpenXR DLLs/SOs get used in the project via MSBuild flags.

## An example use-case

A big one right now is the `openxr_loader`, since [Oculus doesn't currently support the generic openxr_loader for android](https://forums.oculusvr.com/t5/OpenXR-Development/OpenXR-Generic-Loader/td-p/957609), it would be helpful to switch between Oculus and generic with a simple switch.

Similarly, it might be nice to ship a binary that supports deep profiling of StereoKit! This isn't something you'd want to ship in a final build, but could be very helpful to switch to and from for debug and development.

**Comment by maluoi** [comment]
Added in 2671a7f9d40129bf105592767a218cbc9f53d185

**Comment by maluoi** [comment]
This is now available in v0.3.7-preview.6! This can be adjusted by specifying the loader in the .csproj:

Oculus is currently the default, but you can explicitly state this as:
```xml
<SKOpenXRLoader>Oculus</SKOpenXRLoader>
```

For any headset running a _conformant_ OpenXR runtime, use this:
```xml
<SKOpenXRLoader>Standard</SKOpenXRLoader>
```

If you need to add a custom OpenXR loader, you will need to add your loader to `C:\Users\[user]\.nuget\packages\stereokit\0.3.x\android_openxr_loaders`, and specify that name in your SKOpenXRLoader instead.
